tests/aws/services/cloudformation/resources/test_lambda.py

```diff
@@ -239,6 +239,12 @@ def test_lambda_alias(deploy_cfn_template, snapshot, aws_client):
         parameters={"FunctionName": function_name, "AliasName": alias_name},
     )
 
+    invoke_result = aws_client.lambda_.invoke(
+        FunctionName=function_name, Qualifier=alias_name, Payload=b"{}"
+    )
+    assert "FunctionError" not in invoke_result
```

Comment:
q: do we still need this assertion of we are snapshotting the result?

Reply:
The assertion is not strictly when it comes to AWS<->LS parity. However, we commonly clarify the intention for Lambda invokes (see Lambda test suite) because it's too/so easy to overlook a snapshot update, which contains a failing Lambda response (happened twice here).